### PR TITLE
Updated colab_facets.ipynb to make the atlas_url and sprite feature work using base64 data url.

### PR DESCRIPTION
**Pull request by jsiddique**
Problem:
The atlas and sprite feature is missing in the current Colab notebook example.  The atlas_url feature in the current Colab example didn't work directly using local files and regular url.  There is no easy way to pass the image using atlas_url from Colab.  

Solution:
The solution is to read image from the file and convert it into base64 string.  Then create a base64 image url from the string and pass the base64 image url directly into atlas_url. 

Added new dive example showing the atlas_url and sprite feature.  The notebook works in Colab and is able to show the visualizaiton properly.

### CODE REVIEW

**Comment by jsiddique** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2FPAIR-code%2Ffacets%2Fpull%2F232) for more info**.

@googlebot I signed it!

**Comment by jameswex** [comment]
Thanks for putting this together @jsiddique . Could you instead put this atlas example in a new ipynb file in the facets_dive directory? I think it makes sense to have it as a standalone Dive example notebook, separate from the simple one at the top-level directory.

After you do that, if its OK with you I might copy the two external files it depends on to our what-if-tool public cloud bucket so that the URLs they point to are connected to the project itself and update the notebook to point to those new URLs. Also perhaps you could add a comment in the notebook about how the atlas image file was created.

File: facets_dive/colab_dive_atlas_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        "import os\n",
+        "import pandas as pd\n",
+        "\n",
+        "#Download atlas image\n",
+        "img_url = \"https://raw.githubusercontent.com/jsiddique/facets_example/master/sprite_atlas.png\"\n",
+        "img_name = os.path.basename(img_url)\n",
+        "urllib.request.urlretrieve(img_url,img_name)\n",
+        "\n",
+        "#download metadata in a dataframe\n",
+        "df_fmnist = pd.read_csv(\"https://raw.githubusercontent.com/jsiddique/facets_example/master/fmnist.csv\")\n",
```
Comment (jameswex) [inline facets_dive/colab_dive_atlas_demo.ipynb:49]: can you change this to https://storage.googleapis.com/what-if-tool-resources/misc-resources/fmnist.csv?

File: facets_dive/colab_dive_atlas_demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      "source": [
+        "# Display the Dive visualization using atlas and base64 image url.\n",
+        "\n",
+        "import base64\n",
+        "import urllib.request\n",
+        "import os\n",
+        "import pandas as pd\n",
+        "\n",
+        "#Download atlas image\n",
+        "img_url = \"https://raw.githubusercontent.com/jsiddique/facets_example/master/sprite_atlas.png\"\n",
```
Comment (jameswex) [inline facets_dive/colab_dive_atlas_demo.ipynb:44]: can you change this to https://storage.googleapis.com/what-if-tool-resources/misc-resources/fmnist_sprite_atlas.png

File: facets_dive/colab_dive_atlas_demo.ipynb (hunk 1 above)
Comment (jsiddique) [reply facets_dive/colab_dive_atlas_demo.ipynb]: Changed the url.

File: facets_dive/colab_dive_atlas_demo.ipynb (hunk 2 above)
Comment (jsiddique) [reply facets_dive/colab_dive_atlas_demo.ipynb]: Changed the url.

**Comment by jsiddique** [comment]
@jameswex Thanks for the suggestions.  Based on your comment I have reverted my previous commit and rolled back the changes I made to colab_facets.ipynb.  I have created a new standalone colab Dive example notebook in facets_dive directory.  The name of the file is colab_dive_atlas_demo.ipynb

Also added some comments in the notebook about how the atlas image file was created. Let me know if the changes are OK.  

Definitely OK with me to copy the two external files in what-if-tool public cloud bucket. I have updated the notebook to point to those new URLs.

**Review by jameswex — APPROVED** [review]

**Comment by jameswex** [comment]
Thanks again @jsiddique . I also added a link to this notebook in the Facets Dive README

**Pull request merged**